src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/ValidationErrors.feature

```diff
@@ -58,3 +58,84 @@ Scenario: Post a valid Descriptor
   """
   Then the response code is 201
 
+Scenario: Post an invalid body for academicWeeks missing more than one required field
+  When sending a POST request to "data/ed-fi/academicWeeks" with body
```

Comment:
@stephenfuqua Could you verify a portion of the URL? Not sure if this could affect somehow, but I recall one of your earlier remarks suggesting to maintain it as `/ed-fi/academicWeeks`

Reply:
Good point - we need to be consistent with this between files. My preference: do not include "data" since it is not a required path segment in an Ed-Fi API.

Reply:
I tried in Postman without the word "data" in the URL and got a 404 Not Found response.

Reply:
Yes, the URL must have "data" for both DMS and ODS/API. But this is not a requirement of _all_ Ed-Fi API applications. Better to stick with the core requirements here. In the C# step definition, we must combine this path segment with the URL base to construct the real URL. We can inject "data/" into the URL at that point.